### DISCUSSION TITLE
Strange mismatching application update

### DISCUSSION THREAD

**Discussion by alice-mm** [tughi/aggregator-android#62] [Q&A] [answered]
Hi,

I’m using the “old” aggregator (`com.tughi.aggregator`), and the native store (“App Lounge”) of /e/OS (https://e.foundation/) suddenly suggested to me an update for it, except that:

* There is no logo (default Android mascot as thumbnail)
* It now bears the name of the “Next” version: Aggregator Next
* The author is not Tughi but Ştefan Şelariu (not sure that’s the exact same diacritic)

… despite the fact that the package name is indeed `com.tughi.aggregator`.
I tried to apply the update (perhaps I shouldn’t have) and it failed due to a “signature mismatch”.

Is that a legit update, like, an incentive to make people migrate to the “Next” version, or is that a malicious person trying to derive profit from the slight confusion that may come from the coexistence of the two apps by impersonating the authors?

Aurora Store, on the other hand, reports no available update.

# App Lounge screenshots

![Screenshot_20221014-222800_App_Lounge](https://user-images.githubusercontent.com/40308596/195938464-ab748594-a5f5-4cee-97b9-3c54c8443ecb.png)
![Screenshot_20221014-222809_App_Lounge](https://user-images.githubusercontent.com/40308596/195938469-01e1f492-aad0-4d24-9892-09702789721d.png)
![Screenshot_20221014-222820_App_Lounge](https://user-images.githubusercontent.com/40308596/195938471-e199885d-f84d-458d-9fba-fc9d20283382.png)

**Comment by tughi** [comment, marked as answer]
I released the app to F-Droid using the same package name used  by the old app on Google Play.

This was done intentionally because the open source version is intended to replace the old app in the future.

**Reply by alice-mm** [reply]
OK, thanks for the clarification.
I think that the App Lounge fetches stuff from various sources, possibly also including F-droid, hence the confusion I guess.

Guess it’s time for me to check out the import / export features or whatever and start migrating to Next. 😅 But the “preview” version thingy made it sound as if it were too early 😆

And sorry for suspecting Ştefan of being a “malicious person” 😅 Duckduckgo failed to find his GitHub profile and only showed me out-of-topic stuff, but afterwards I saw his name in discussions around here.

**Reply by tughi** [reply]
Yes... That's only my name... 😉
